core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -242,6 +242,7 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
           abortResults <- Future.traverse(workflowRecs) { workflowRec =>
             Future.successful(workflowRec.externalId).zip(executionServiceCluster.abort(workflowRec, petUserInfo))
           }
+          // TODO: should this also mark the workflows as Aborted in the Rawls db???
```

Comment:
I don't think so, it could end up being a lie if Rawls tells Cromwell to abort, and Cromwell for some reason does not do so.

If everything works as expected the status will get picked up from Cromwell automatically later.